### DISCUSSION TITLE
downloading Pyodive takes too much time, Can I deploy it locally?

### DISCUSSION THREAD

**Discussion by MotokaJuu** [pyscript/pyscript#1791] [Q&A]
downloading Pyodive takes too much time, about 10 min.

**Comment by WebReflection** [comment]
You can specify an interpreter field in the py-config and pass a fully qualified URL but bear in mind you’ll need also all your packages in it, among many other files.

I think you tested in a bad time as nobody here waits more than few milliseconds to get pyodide from a CDN. It does take its time to bootstrap but if that’s the issue your are after, I think you are on old mobile or a very outdated machine. You can try mpy-script there instead, and see the difference.

I’m also assuming you are talking on current PyScript RC2, not “classic” PyScript.

**Reply by JeffersGlass** [reply]
And if for some really specific reason you *do* need to download Pyodide locally, check out the [instructions from Pyodide](https://pyodide.org/en/stable/usage/downloading-and-deploying.html#github-releases) on downloading a release and it’s associated bundles Python packages
